src/components/bookList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Book from './book';
+
+const BooksList = () => {
+  const bookList = [
+    {
+      id: 1,
+    },
+    {
+      id: 2,
+    },
+  ];
+  const list = bookList.map((item) => (<li key={item.id}><Book /></li>));
+  return (
+    <ul>
+      {list}
+    </ul>
```

Comment:
- [ ] The form for adding a book is supposed to be rendered below the list of books in this component. Kindly import the form component and render it below the ul here. 👍